core/src/main/java/io/grpc/internal/GrpcUtil.java

```diff
@@ -39,6 +39,8 @@
 import com.google.common.base.Joiner;
 import com.google.common.base.Preconditions;
 import com.google.common.base.Splitter;
+import com.google.common.collect.BiMap;
```

Comment:
@madongfly, should we use BiMap in core/? Would that disappoint some of our Android users?